### DISCUSSION TITLE
Drag Handle extension

### DISCUSSION THREAD

**Discussion by laowangdefangjian** [ueberdosis/tiptap#5637] [General Questions] [answered]
Could you please enrich this extension and add a menu feature?
Drag Handle extension

**Comment by kart-c** [comment, marked as answer]
By Menu, if you mean that you want to open a popup when the drag handle is clicked this can be achieved by adding your own menu as per requirement.
e.g, since I am using React, I can do the following 
```jsx
<DragHandle  editor={editor}>
  <Button onClick={openPopup}>
    <DragIcon />
  </Button>
  <PopupContent /> // only shown when popup is opened
</DragHandle>
```

**Reply by laowangdefangjian** [reply]
I'm not using React; I'm using Vue 3. I don't know how to click and drag the handle to open the settings menu and configure the corresponding component based on the official example.

**Reply by kart-c** [reply]
I haven't used Vue extensively, but I would if you can share a sandbox, I can try and help.
